Modules/cjkcodecs/multibytecodec.c

```diff
@@ -895,6 +897,102 @@ _multibytecodec_MultibyteIncrementalEncoder_encode_impl(MultibyteIncrementalEnco
     return encoder_encode_stateful(STATEFUL_ECTX(self), input, final);
 }
 
+/*[clinic input]
+_multibytecodec.MultibyteIncrementalEncoder.getstate
+[clinic start generated code]*/
+
+static PyObject *
+_multibytecodec_MultibyteIncrementalEncoder_getstate_impl(MultibyteIncrementalEncoderObject *self)
+/*[clinic end generated code: output=9794a5ace70d7048 input=4a2a82874ffa40bb]*/
+{
+    const char *pendingbuffer;
+    Py_ssize_t pendingsize;
+
+    if (self->pending != NULL) {
+        pendingbuffer = PyUnicode_AsUTF8AndSize(self->pending, &pendingsize);
+        if (pendingbuffer == NULL) {
+            return NULL;
+        }
+
+        return _PyLong_FromByteArray((const unsigned char *)pendingbuffer,
+                                     pendingsize, PY_LITTLE_ENDIAN, 0);
+    }
+
+    return _PyLong_Zero;
```

Comment:
`_Py_INCREF(_PyLong_Zero)` is required before return.
Or you can use `return PyLong_FromLong(0);`.

Reply:
Thanks.